src/write_gcm.c

```diff
@@ -2367,11 +2372,12 @@ static int wg_write(const data_set_t *ds, const value_list_t *vl,
   pthread_mutex_lock(&queue->mutex);
   // Backpressure. If queue is backed up then something has gone horribly wrong.
   // Maybe the queue processor died.
-  size_t abnormal_limit = 5 * MAX_QUEUE_SIZE;
-  if (queue->size > abnormal_limit) {
+  if (queue->size > QUEUE_DROP_SIZE) {
+    DEBUG("write_gcm: Dropping data point because queue has size %zd",
```

Comment:
Should we drop the oldest point rather than the newest point in these situations?  Maybe there's something "bad" about the old point and that's why they're there?